e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -725,51 +725,41 @@ describe("scenarios > dashboard > tabs", () => {
     cy.findAllByTestId("tab-button-input-wrapper").eq(2).findByText("Tab 3");
 
     // Prior to this bugfix, tab containing this text would be too long to drag to the left of either of the other tabs.
-    const longName =
-      "This is a really, really, really, really, really long tab name";
+    const longName = "This is a really really long tab name";
```

Comment:
Anecdotally a slightly shorter tab name helped prevent flakes in the mouse movements below, but is still long enough to test this edge case.